src/analysis/connectivity/overall_scores.sql

```diff
@@ -23,7 +23,7 @@ CREATE TABLE generated.neighborhood_overall_scores (
 INSERT INTO generated.neighborhood_overall_scores (
     category, score_original, score_normalized, human_explanation
 )
-SELECT  'People',
+SELECT  'people',
```

Comment:
Is the category field now misnamed? Should it just be a `score_identifier` or `score_name` instead? It seems to me that a category would be something like `opportunity`, not `opportunity_employment` which is a specific score or metric?

Also, since we'll be saving category, human_explanation, and any other metadata separately, the human_explanation field here can be removed, and also be removed from the neighborhood_score_inputs table. I'm ok with leaving them in for now if we want to wait to remove until we setup the replacement source for this information.